src/containers/Round/round.js

```diff
@@ -46,9 +46,23 @@ const Round = ({
   }
 
   useEffect(() => {
-    checkForMatch()
+    console.log('completed words length in use effect', completedWords.length)
+   if (column1 !== null && column2 !== null) {
+      console.log('hi')
+      checkForMatch()
+    }
   }, [column1, column2])
 
+  useEffect(() => {
+    if (completedWords.length === 5) {
+      let current = currentRound
+      let updatedRound = currentRound++
+      increaseRound(currentRound)
+      console.log('after increase round', currentRound)
+      buildroundUpCompletedCards()
+    } 
+  }, [completedWords])
```

Comment:
Great new useEffect to check when conpletedWords is updated to update the current round!

Reply:
shoot I didn't realize I left those variables in there that we don't need. I'll update and re-push shortly!